src/data/data.js

```diff
@@ -0,0 +1,3 @@
+export const API_CONTINENT = 'https://disease.sh/v3/covid-19/countries';
+
+export const GET_CONTNINENT = 'GET_CONTINENT';
```

Comment:
- [Optional] Instead of using an extra file, you should have added this piece of code directly in Api.js file. It's difficult to debug a project with so many file structures. And defining the GET_CONTINENT should have followed the duck pattern.